### PR TITLE
Project review

### PR DESCRIPTION
**Pull request by samuelmoiwa**
## Solution description

***CSS:*** Check for code correctness in the CSS style files
***HTML*** Check the Html code for HTML semantic tags, and code correctness

## Address affected and ensured
* HTML file
* CSS file
---

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+              <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+              <li><a class="nav-link" href="#pricing">Pricing</a></li>
+            </ul>
+          </nav>
+        </header>
+      
+  </head>
+
+
+  <body>
```
Comment (samuelmoiwa) [inline index.html:44]: I think the code will look nicer and more readable if the unnecessary white spaces are removed 😊.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 200px;
+    margin-top: 50px;
+}
+
+#hero > h2 {
```
Comment (samuelmoiwa) [inline style.css:90]: I think it'll be more professional to use class(.) for CSS selector properties than to use ID(#)👌

File: index.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+
+    <footer>
+      <ul>
+        <li><a href="#">Privacy</a></li>
+        <li><a href="#">Terms</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+      <span>Copyright 2016, Original Trombones</span>
+    </footer>
```
Comment (samuelmoiwa) [inline index.html:153]: I think it'll be best practice to put all HTML tags, including the footer tag, inside the main body tag of the code. 👌 It makes the web page more informative and adaptable, allowing browsers and search engines to better interpret content.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly ensure that the student has a gitignore file in the Project". Remember that this is the first thing to check in every project.
 - [ ] Please make sure that the student has installed Linters. any project without must be invalid but since is a trial review you have to mention it.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```
Comment (samuelmoiwa) [inline index.html:1]: The work is looking perfect so far.
I think there are a few key configurations you are still missing.
In other for this project to follow the proper git flow, I think you need to set up a few files.

-  add **.gitignore** to your project
-  **Linters**  [ link to the linters life, find setup instructions](https://github.com/microverseinc/linters-config)

**Review by Qoosim — APPROVED** [review]
## **Status: Approved** :tada: 

Hi @samuelmoiwa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  Descriptive pull request message. :heavy_check_mark: 
-  Professional Documentation of work. :heavy_check_mark: 
-  Professional README file. :heavy_check_mark: 
-  Meaningful commit messages. :heavy_check_mark: 


Got

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**